### ISSUE TITLE
Was the flipping along the x and z axes taken into account when reading the 3D image?

### ISSUE THREAD

**Issue by SangbaekYoo** [eigenvivek/DiffDRR#375]
Hello, I have a question regarding the usage of DiffDRR.

When working with NIfTI files, the image data is typically stored in the order of (Z, Y, X) axes. However, I noticed that in DiffDRR, when rendering images using PyTorch, the data seems to be interpreted with the first dimension as the x-axis and the third dimension as the z-axis.

```python
# %% ../notebooks/api/01_renderers.ipynb 8
def _get_alphas(source, target, dims, eps, filter_intersections_outside_volume):
    """Calculates the parametric intersections of each ray with the planes of the CT volume."""
    # Parameterize the parallel XYZ planes that comprise the CT volumes
    alphax = torch.arange(dims[0] + 1).to(source)
    alphay = torch.arange(dims[1] + 1).to(source)
    alphaz = torch.arange(dims[2] + 1).to(source)

    # Calculate the parametric intersection of each ray with every plane
    sx, sy, sz = source[..., 0:1], source[..., 1:2], source[..., 2:3]
    tx, ty, tz = target[..., 0:1], target[..., 1:2], target[..., 2:3]
    alphax = (alphax.expand(len(source), 1, -1) - sx) / (tx - sx + eps)
    alphay = (alphay.expand(len(source), 1, -1) - sy) / (ty - sy + eps)
    alphaz = (alphaz.expand(len(source), 1, -1) - sz) / (tz - sz + eps)
    alphas = torch.cat([alphax, alphay, alphaz], dim=-1)
```

I would like to confirm whether this axis interpretation is intentional, or if I might be misunderstanding the coordinate convention used in DiffDRR.

Thank you in advance for your clarification.

Best regards,
Sangbaek Yoo
